etcdserver/server.go

```diff
@@ -161,13 +161,15 @@ type EtcdServer struct {
 	cluster *cluster
 
 	store store.Store
-	kv    dstorage.KV
+	kv    dstorage.ConsistentWatchableKV
 
 	stats  *stats.ServerStats
 	lstats *stats.LeaderStats
 
 	SyncTicker <-chan time.Time
 
+	// consistent index of the applying entry used to initialize v3 storage
+	consistIndex types.Uint64Value
```

Comment:
also i feel we should change the `Get` interface to `GetIndex` since `Get` is too generic. A unit64Value should not implement GetConistentIndex interface for example.